.github/workflows/scripts/test_bindings.rb

```diff
@@ -0,0 +1,153 @@
+  config.debugging=true
+end
+
+
+@artifacts_api = PulpcoreClient::ArtifactsApi.new
+@filerepositories_api = PulpFileClient::RepositoriesFileApi.new
+@repoversions_api = PulpFileClient::RepositoriesFileVersionsApi.new
+@filecontent_api = PulpFileClient::ContentFilesApi.new
+@filedistributions_api = PulpFileClient::DistributionsFileApi.new
+@filepublications_api = PulpFileClient::PublicationsFileApi.new
+@fileremotes_api = PulpFileClient::RemotesFileApi.new
+@tasks_api = PulpcoreClient::TasksApi.new
+@uploads_api = PulpcoreClient::UploadsApi.new
+
+
+def monitor_task(task_href)
+    # Polls the Task API until the task is in a completed state.
+    #
+    # Prints the task details and a success or failure message. Exits on failure.
+    #
+    # Args:
+    #    task_href(str): The href of the task to monitor
+    #
+    # Returns:
+    #     list[str]: List of hrefs that identify resource created by the task
+    completed = []
+    task = @tasks_api.read(task_href)
+    until ["completed", "failed", "canceled"].include? task.state
+      sleep(2)
+      task = @tasks_api.read(task_href)
+    end
+    if task.state == 'completed'
+      task.created_resources
+    else
+      print("Task failed. Exiting.\n")
+      exit(2)
+    end
+end
+
+def content_range(start, finish, total)
+  finish = finish > total ? total : finish
+  "bytes #{start}-#{finish}/#{total}"
+end
+
+def upload_file_in_chunks(file_path)
+    # Uploads a file using the Uploads API
+    #
+    # The file located at 'file_path' is uploaded in chunks of 200kb.
+    #
+    # Args:
+    #     file_path (str): path to the file being uploaded to Pulp
+    #
+    # Returns:
+    #     upload object
+    response = ""
+    File.open(file_path, "rb") do |file|
+      total_size = File.size(file)
+      upload_data = PulpcoreClient::Upload.new({size: total_size})
+      response = @uploads_api.create(upload_data)
+      upload_href = response.pulp_href
+      chunksize = 200000
+      offset = 0
+      sha256 = Digest::SHA256.new
+      until file.eof?
+        chunk = file.read(chunksize)
+        sha256.update(chunk)
+        begin
+          filechunk = Tempfile.new('fred')
+          filechunk.write(chunk)
+          filechunk.flush()
+          actual_chunk_size = File.size(filechunk)
+          response = @uploads_api.update(content_range(offset, offset + actual_chunk_size -1, total_size), upload_href, filechunk)
+          offset += actual_chunk_size -1
+        ensure
+          filechunk.close
+          filechunk.unlink
+        end
+      end
+      upload_commit_response = @uploads_api.commit(upload_href, {sha256: sha256.hexdigest})
+      created_resources = monitor_task(upload_commit_response.task)
+      @artifacts_api.read(created_resources[0])
+    end
+end
+
+
+artifact = upload_file_in_chunks(File.join(ENV['GITHUB_WORKSPACE'], 'README.rst'))
+
+# Create a File Remote
+remote_url = 'https://repos.fedorapeople.org/pulp/pulp/demo_repos/test_file_repo/PULP_MANIFEST'
```

Comment:
```suggestion
remote_url = 'https://fixtures.pulpproject.org/file/PULP_MANIFEST'
```

Reply:
I'm going to let https://github.com/pulp/pulpcore/pull/1049 merge first and then copy the files again